services/pypi/pypi-framework-versions.service.js

```diff
@@ -0,0 +1,111 @@
+import PypiBase from './pypi-base.js'
+import { sortPypiVersions, parseClassifiers } from './pypi-helpers.js'
+
+const documentation = `
+<p>
+  This service currently support the following Frameworks: <br/>
+  <strong>AWS CDK</strong>,
+  <strong>Django</strong>,
+  <strong>Django CMS</strong>,
+  <strong>JupyterLab</strong>,
+  <strong>Odoo</strong>,
+  <strong>Plone</strong>,
+  <strong>Wagtail</strong>,
+  <strong>Zope</strong>.
+</p>
+`
+
+const frameworkNameMap = {
+  awscdk: {
+    name: 'AWS CDK',
+    classifier: 'AWS CDK',
+  },
+  django: {
+    name: 'Django',
+    classifier: 'Django',
+  },
+  djangocms: {
+    name: 'Django CMS',
+    classifier: 'Django CMS',
+  },
+  jupyterlab: {
+    name: 'JupyterLab',
+    classifier: 'Jupyter :: JupyterLab',
+  },
+  odoo: {
+    name: 'Odoo',
+    classifier: 'Odoo',
+  },
+  plone: {
+    name: 'Plone',
+    classifier: 'Plone',
+  },
+  wagtail: {
+    name: 'Wagtail',
+    classifier: 'Wagtail',
+  },
+  zope: {
+    name: 'Zope',
+    classifier: 'Zope',
+  },
+}
+
+export default class PypiFrameworkVersion extends PypiBase {
+  static category = 'platform-support'
+
+  static route = {
+    base: 'pypi/frameworkversions',
+    pattern:
+      ':frameworkName(awscdk|django|djangocms|jupyterlab|odoo|plone|wagtail|zope)/:packageName*',
+  }
+
+  static examples = [
+    {
+      title: 'PyPI - Framework Version',
```

Comment:
I don't particularly want to list a separate example for each framework but it would be nice to give more of an indication that all the framework classifiers work here. Maybe we call it something like "PyPI - Versions from Framework Classifiers"? (naming things is hard)